sci-mathematics/cgal/cgal-5.1.ebuild

```diff
@@ -0,0 +1,78 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake flag-o-matic
+CMAKE_BUILD_TYPE=Release
```

Comment:
Why?

Reply:
Oof, I thought I had removed `flag-o-matic`
done.

Reply:
That was not what I meant, and that's not the line where I left this comment. What's wrong with the default build type of "Gentoo"?

Reply:
Then patch that check away :)

Reply:
…and send it upstream

Reply:
done.

Reply:
I think you forgot `git add`

Reply:
Hmmm, no it should be there.

Reply:
Ah, it's not a new file? I see.